specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -392,6 +392,149 @@
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query.",
+            "schema": {
+              "$ref": "#/definitions/PutSubscriptionResponse"
+            }
+          },
+          "202": {
+            "description": "Accepted. Subscription creation is in progress.",
+            "headers": {
+              "Location": {
+                "description": "GET this URL to retrieve the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "integer"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PutSubscriptionRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      },
+      "get": {
+        "description": "Get Subscription.",
+        "operationId": "Subscription_GetSubscription",
+        "x-ms-examples": {
+          "getSubscription": {
+            "$ref": "./examples/GetSubscription.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Subscription details.",
+            "schema": {
+              "$ref": "#/definitions/PutSubscriptionResponse"
+            }
+          },
+          "202": {
+            "description": "Subscription creation is in progress.",
+            "headers": {
+              "Location": {
+                "description": "GET this URL to retrieve the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "integer"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Subscription/OperationResults/{operationId}": {
```

Comment:
@allenjzhang it's unusual for RPs to declare the operationresults endpoint in swagger because it's generally not an API that a customer would want to call directly. Please could you comment on whether this should be present?

Reply:
taken out for now - checked for operationResults API in few swaggers (compute, network, storage) and did not find it.
@allenjzhang - let us know if we need this, else please review and merge the PR.

Reply:
Some RP like Media Services does have Operations API. Though we would like to standardize to the [async azure pattern](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/async-operations#azure-asyncoperation-request-and-response)

@sriramvu, the title of this PR says DO NOT MERGE. So I added label as well. If you are ready, please update title and remove the label and I can do the merge. Thanks!

Reply:
1) reg operationResults API: we do have async azure pattern, this in particular is for the one that comes back in location header which has operationResults in the uri - yes, seeing the same in media services swagger - please recommend whether this should be included or not in this PR as this is the not the API we expect in general any customer to explicitly use it.
2) sure, will update the title and remove the tag once we are ready - Thank you.